src/main/java/core/basesyntax/model/Shape.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+public abstract class Shape implements Figure {
```

Comment:
implements interfaces from comments above
```suggestion
public abstract class Figure {
```